netty/src/main/java/io/grpc/netty/AbstractHttp2Headers.java

```diff
@@ -26,6 +26,11 @@
 abstract class AbstractHttp2Headers implements Http2Headers {
 
   @Override
+  public Iterator<CharSequence> valueIterator(CharSequence name) {
```

Comment:
Do we have to change Bazel repositories as well?

Reply:
Done